draft-ietf-tls-esni.md

```diff
@@ -324,25 +339,20 @@ The payload MUST have the following structure:
 
 ~~~~
     struct {
+       opaque config_id<0..255>
        ECHCipherSuite cipher_suite;
-       opaque config_id<0..255>;
```

Comment:
Why change the wire format of the ECH extension? We might end up changing the type of `config_id` anyway, in which case the point is moot. However, for now, I think we should keep the extension parameters in the same order.

Reply:
Reverted!